docs/eggs/publishing-a-module.md

```diff
@@ -80,17 +80,21 @@ on:
 
 jobs:
   publish-egg:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

let the runner be latest

Reply:
> 
> 
> Hey @GJZwiers, thanks for the update :)
> 
> I left a few suggestions, just update the pr and we can merge

Thanks! I have made the suggested changes